_posts/2020-07-22-cockpit-224.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Cockpit 224
```

Comment:
How about devoting this blog post entirely to cockpit-podman 20, and drop the header prefixes? Or at least mention cockpit-podman in title:?

Reply:
Yeah, makes sense!